org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/chrome/ChromeRunDAPDebugDelegate.java

```diff
@@ -106,11 +108,24 @@ static File findDebugAdapter() {
 			Display.getDefault().asyncExec(() -> ErrorDialog.openError(Display.getDefault().getActiveShell(), "Debug error", e.getMessage(), errorStatus)); //$NON-NLS-1$
 		}
 		return null;
-		
 	}
 
-	private String findChromeLocation() {
-		String res = InitializeLaunchConfigurations.which("chromium-browser");
+	static String findChromeLocation(ILaunchConfiguration configuration) {
+		String res = null;
+		try {
+			if (configuration.getAttribute(RUNTIME_EXECUTABLE, "no runtime executable set").equals(CHROMIUM)) {
```

Comment:
In this case, let the default value be `chromium-browser` so we save a null check after that.

Reply:
Good call